cmd/everoute-agent/config.go

```diff
@@ -62,10 +70,10 @@ func getAgentConfig() (*agentConfig, error) {
 	return &agentConfig, nil
 }
 
-func getDatapathConfig() (*datapath.Config, error) {
+func getDatapathConfig() (*datapath.Config, *datapath.AgentConf, error) {
```

Comment:
Could we merge two configs ?

Reply:
use config to store static config
use info to store the info from apiserver or somewhere